### ISSUE TITLE
Unsupported level format message is confusing

### ISSUE THREAD

**Issue by rexim** [tsoding/nothing#1188]
Introduced in #1166

## Steps to reproduce

1. Start the game
1. Press `N`
1. Press `TAB`
1. Press `S`
1. Give a filename
1. Press `Enter`
1. Quit the game
1. Open the newly create level file
1. Remove the line with version
1. Start the game

## Observed

The new level does not appear in the menu. The game printed in the logs:

```
[INFO] Level `fffda5` with unsupported version New Level rejected, expected 1
```

## Expected:

A less confusing message telling us that we are using an unsupported level format.

**Comment by rexim** [comment]
It looks confusing probably because of the `Level fffda5` part, because as a user of the problem I don't really have level `fffda5` and I have no idea what it is talking about.

**Comment by zhiayang** [comment]
the issue comes from the fact that the level file is supposed to start with the version number. if there's no version number, then the error message treats the first line as the version -- which is the level name, hence "unsupported version New Level". then the next line, which is supposed to be the level name but is actually the rect data, is read as the name. sorry for writing a fairytale

as i said in ~issue tracker~ chat, this is kinda weird because there's only 2 levels right now, and all future levels will have the version on the first line, which would make this messsage non-confusing.

**Comment by rexim** [comment]
Resolved in #1192
